app/_src/guides/gateway-delegated.md

```diff
@@ -0,0 +1,197 @@
+---
+flowchart LR
+  subgraph Kong Gateway 
+    gw0(/ :80)
+  end
+  demo-app(demo-app :5000)
+  redis(redis :6379)
+  gw0 --> demo-app 
+  demo-app --> redis
+{% endmermaid %}
+
+## Prerequisites
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+
+## Install Kong ingress controller 
+
+Follow the steps on the [Kong docs website](https://docs.konghq.com/kubernetes-ingress-controller/latest/get-started/) to install the ingress controller.
+
+{% warning %}
+The Kubernetes cluster needs to support LoadBalancer for this to work.
+This may not be the case if the Kubernetes cluster is running locally with `kind` or `k3d`.
+{% endwarning %}
+
+## Enable sidecar injection on the `kong` namespace
+
+The Kong Ingress controller was installed outside the mesh.
+For it to work as a delegated gateway restart it with [sidecar injection enabled](/docs/{{ page.version }}/production/dp-config/dpp-on-kubernetes/):
+
+Add the label:
+```shell
+kubectl label namespace kong kuma.io/sidecar-injection=enabled
+```
+
+Restart both the controller and the gateway to leverage sidecar injection:
+```shell
+kubectl rollout restart -n kong deployment kong-gateway kong-controller
+```
+
+Wait until pods are fully rollout and look at them:
+```shell
+kubectl get pods -n kong
+```
+
+It is now visible that both pods have 2 containers, one for the application and one for the sidecar.
+```shell
+NAME                              READY   STATUS    RESTARTS      AGE
+kong-controller-675d48d48-vqllj   2/2     Running   2 (69s ago)   72s
+kong-gateway-674c44c5c4-cvsr8     2/2     Running   0             72s
+```
+
+Verify the gateway still works:
+```shell
+curl -i $PROXY_IP
+```
+
+outputs that there are no routes defined:
+```shell
+HTTP/1.1 404 Not Found
+Date: Fri, 09 Feb 2024 15:25:45 GMT
+Content-Type: application/json; charset=utf-8
+Connection: keep-alive
+Content-Length: 103
+X-Kong-Response-Latency: 0
+Server: kong/3.5.0
+X-Kong-Request-Id: e7dfe659c9e46639a382f82c16d9582f
+
+{
+  "message":"no Route matched with those values",
+  "request_id":"e7dfe659c9e46639a382f82c16d9582f"
+}%
+```
+
+## Add a route to our `demo-app`
+
+Patch our gateway to allow routes in any namespace:
+
+```shell
+kubectl patch --type=json gateways.gateway.networking.k8s.io kong -p='[{"op":"replace","path": "/spec/listeners/0/allowedRoutes/namespaces/from","value":"All"}]'
+```
+
+Add the route in our `kuma-demo` namespace which binds to the gateway `kong` defined in the `kong` namespace:
+```shell
+echo "
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: demo-app
+  namespace: kuma-demo
+spec:
+  parentRefs:
+  - name: kong
+    namespace: default
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /
+    backendRefs:
+    - name: demo-app
+      kind: Service
+      port: 5000 
+" | kubectl apply -f -
+```
```

Comment:
It might be worth calling out that this is adding `HTTPRoute` to kong and not kuma (kinda 1% guessing here, but I think this is the case seeing as we have delegated all this to the 3rd party gateway). I had a double take for a second and was like "oooooooh, its not a `MeshHTTPRoute` its a `HTTPRoute` that must be a kong gateway thing". You kinda hint at this when I read it back again, but might be worth adding a little more?

Also, at first I thought there was a mistake in the code here as the `parentRef` uses the `default` namespace not the `kong` one as you mention. This might just be my misunderstanding, so this might be a query just for me, but also might be worth explaining that in the text, not sure up to you.